tests/server/test_links.py

```diff
@@ -17,7 +17,7 @@
 BUILD_37 = 37
 BUILD_38 = 38
 
- = "GRCh37"
+BEACON_BUILD_37 = "GRCh37"
```

Comment:
No, now the other tests of this page will fail!

Reply:
ah no, you added a new constant

Reply:
I dont think so - it would if I changed `BUILD` - but let's see. This is an exciting show is it not!

Reply:
I think it's lunchtime